packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -34,7 +28,7 @@ export function ContributeSection({
     trackEvent({
       event_name: 'start submit article',
       feed_item_title: 'Submit article',
-      extra: JSON.stringify({ has_access: canSubmitArticle }),
+      extra: JSON.stringify({ has_access: true }),
```

Comment:
is this correct as always true? and if yes, why don't we remove it entirely?

Reply:
This won't get triggered until the user will have the proper access. So it will always be true, yes, once the user is eligible.

Reply:
In the past, this was always based on the flagsmith value which always `true` since we released it to everyone. But maybe what we should have done is verify whether the user is eligible.

This seems like we had the wrong value all along as it is always true.

Reply:
I have now moved the tracking on the place where we identify whether the user is eligible.
https://github.com/dailydotdev/apps/pull/2160/commits/2ba26b35d112c224854ec2cdfe8efe1a2d17f548

Reply:
After checking the API, this seems to be adjusted on the go. So we were getting the right values for this flag.

Since we will be removing flagsmith, and calls to feature flags are done directly to GB, we won't be able to do overrides anymore.

We have two options:
- The first is to go with the commit mentioned above.
- The second one is we will utilize the targeting conditions offered by GB since the value is based on the reputation which is found on the user's table which we can access from there.

Would love to hear what you guys think.

Reply:
Good point, I totally missed that part. Yep, we can do the user state. Then for outdated extension users, we will just keep the API logic as it is where it checks whether the user should have it.